### ISSUE TITLE
HW 7 bonus

### ISSUE THREAD

**Issue by florian-rabe** [intigabi/SaDS#6]
Can you add a little bit about how you obtained the solution to 7.4?

**Comment by intigabi** [comment]
Yes ofc, I used a brain command called "try_and_guess_the_answer_and_hope_its_correct_if_someone_checks_it.exe".

Highly unlikely that I got the answer correctly. Sorry if I wasted your time.

**Comment by florian-rabe** [comment]
OK, then no bonus points for you.
